build.sh

```diff
@@ -25,8 +26,10 @@ for i in "$@"; do
         -t|--target) TARGET="$2"; shift ;;
         -c|--configuration) CONFIGURATION="$2"; shift ;;
         -v|--verbosity) VERBOSITY="$2"; shift ;;
-        -d|--dryrun) DRYRUN=true ;;
+        -d|--dryrun) DRYRUN="-dryrun" ;;
         --version) SHOW_VERSION=true ;;
+        --) shift; SCRIPT_ARGUMENTS+=("$@"); break ;;
```

Comment:
@patriksvensson Yep, `"$@"` should preserve the separation between the arguments. Something like `1 2 3` should get forwarded as `1 2 3` and not `"1 2 3"`.